controllers/policyset/policyset_controller.go

```diff
@@ -234,44 +241,56 @@ func (r *PolicySetReconciler) processPolicySet(ctx context.Context, plcSet *poli
 	return needsUpdate
 }
 
-// getStatusMessage returns a message listing disabled, deleted and pending policies
-func getStatusMessage(disabledPolicies []string, pendingPolicies []string, deletedPolicies []string) string {
-	statusMessage := "All policies are reporting status"
+// getStatusMessage returns a message listing disabled, deleted and policies with no status
+func getStatusMessage(
+	disabledPolicies []string,
+	unknownPolicies []string,
+	deletedPolicies []string,
+	pendingPolicies []string,
+) string {
+	statusMessage := ""
 	separator := ""
+	allReporting := true
+
+	if len(pendingPolicies) > 0 {
+		allReporting = false
+		statusMessage = fmt.Sprintf(statusMessage+separator+
+			"Policies awaiting pending dependencies: %s", strings.Join(pendingPolicies, ", "))
```

Comment:
This is easier to read in my opinion
```suggestion
		statusMessage += seperator + fmt.Sprintf("Policies awaiting pending dependencies: %s", strings.Join(pendingPolicies, ", "))
```